extensions/wikia/AchievementsII/css/leaderboard_oasis.scss

```diff
@@ -67,13 +67,13 @@
 	 		width: 50px;		
 		}
 		.bronze span {
-			background: url(/extensions/wikia/AchievementsII/images/bronze_badge_blank.png); /* $wgCdnStylePath */
+			background: url(/extensions/wikia/AchievementsII/images/bronze_badge_blank.png); /* $wgResourceBasePath */
 		}
 		.silver span {
-			background: url(/extensions/wikia/AchievementsII/images/silver_badge_blank.png); /* $wgCdnStylePath */
+			background: url(/extensions/wikia/AchievementsII/images/silver_badge_blank.png); /* $wgResourceBasePath */
 		}
 		.gold span {
-			background: url(/extensions/wikia/AchievementsII/images/gold_badge_blank.png); /* $wgCdnStylePath */
+			background: url(/extensions/wikia/AchievementsII/images/gold_badge_blank.png); /* $wgResourceBasePath */
```

Comment:
I'd double check if these comments are actually still required to perform a URL rewrite on AssetsManager SASS renderer.